code/src/java/pcgen/gui2/PCGenFrame.java

```diff
@@ -1827,7 +1829,7 @@ private void showMatureDialog(String text)
 
 		aFrame.getContentPane().setLayout(new BorderLayout());
 		aFrame.getContentPane().add(jPanel1, BorderLayout.NORTH);
-		aFrame.getContentPane().add(htmlPanel, BorderLayout.CENTER);
+//		aFrame.getContentPane().add(htmlPanel, BorderLayout.CENTER);
```

Comment:
Should we remove this rather than comment out?

Reply:
Neither. It shouldn't have been commented out :)